### PR TITLE
Adding logging 6.1 build

### PR DESCRIPTION
**Pull request by briandooley**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-logging-docs-6.1.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 400m
+      memory: 800Mi
+tests:
+- as: validate-asciidoc
+  steps:
+    env:
+      DISTROS: logging-docs
```
Comment (aireilly) [inline ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-logging-docs-6.1.yaml:18]:
```suggestion
      DISTROS: openshift-docs
```

File: ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-logging-docs-6.1.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - ref: openshift-docs-preview-comment-pages
+    - ref: openshift-docs-asciidoctor
+    - ref: openshift-docs-lint-topicmaps
+    - ref: openshift-docs-jira-links
+    - ref: openshift-docs-vale-review
+- as: validate-portal
+  steps:
+    env:
+      BUILD: build_for_portal.py
+      DISTROS: logging-docs
```
Comment (aireilly) [inline ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-logging-docs-6.1.yaml:31]:
```suggestion
      DISTROS: openshift-docs
```

**Comment by aireilly** [comment]
squash your commits

**Comment by briandooley** [comment]
commits have been squashed

File: ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-logging-docs-6.1.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 400m
+      memory: 800Mi
+tests:
+- as: validate-asciidoc
+  steps:
+    env:
+      DISTROS: openshift-docs
```
Comment (aireilly) [inline ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-logging-docs-6.1.yaml:18]:
```suggestion
      DISTROS: openshift-enterprise
```
oops

File: ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-logging-docs-6.1.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    - ref: openshift-docs-preview-comment-pages
+    - ref: openshift-docs-asciidoctor
+    - ref: openshift-docs-lint-topicmaps
+    - ref: openshift-docs-jira-links
+    - ref: openshift-docs-vale-review
+- as: validate-portal
+  steps:
+    env:
+      BUILD: build_for_portal.py
+      DISTROS: openshift-docs
```
Comment (aireilly) [inline ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-logging-docs-6.1.yaml:31]:
```suggestion
      DISTROS: openshift-enterprise
```

**Review by aireilly — APPROVED** [review]

**Pull request merged**